Models/tournaments/matchup.ts

```diff
@@ -88,6 +88,9 @@ export class Matchup extends BaseEntity {
     @OneToMany(() => MatchupMessage, message => message.matchup)
         messages?: MatchupMessage[] | null;
 
+    @Column({ type: "varchar", length: 15, nullable: true, select: false })
+        ip?: string | null;
```

Comment:
```suggestion
    @Column({ type: "varchar", length: `http://255.255.255.255:65565`.length, nullable: true, select: false })
        baseUrl?: string | null;
```
be careful with `select: false`, it is error-prone. the IP is internal and not confidential info, so if that's your concern, you can remove that.